# AgentState.msg
int32 id
float32[] state